src/app/modules/library/select-all-items-checkbox/select-all-items-checkbox.component.html

```diff
@@ -1,7 +1,7 @@
 <mat-checkbox
   color="primary"
-  [checked]="selectedAllItems"
-  [indeterminate]="selectedSomeItems"
+  [checked]="status == 'all'"
+  [indeterminate]="status == 'some'"
```

Comment:
These equality checks can be changed to ===

Reply:
Good catch. Made the change.